libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts

```diff
@@ -201,3 +217,50 @@ const convertAlert = (
 
     return undefined;
 };
+
+type SavedMetadata = Omit<IAutomationMetadataObject["metadata"], "filters"> & { filters?: number[] };
+
+function convertFiltersAndMetadata(
+    metadata: SavedMetadata | null | undefined,
+    alert: IAutomationAlert | undefined,
+) {
+    const filtersIndexes = metadata?.filters ?? [];
+    const filtersIds: string[] = [];
+    const filters = alert?.execution.filters.map((filter, index) => {
+        if (filtersIndexes.includes(index) && isPositiveAttributeFilter(filter)) {
+            const localIdentifier = uuidv4();
```

Comment:
Is this correct? Filter local ids should likely match with attribute local identifiers or am I wrong?

Reply:
+ it's non-deterministic, so every time you get the same automation from backend, the MD object will be different.

Reply:
Local identifiers are not converted and store on server, im not sure if its ok, but server spec not qallow to do it now